packages/typia-validator/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@hono/typia-validator",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

```suggestion
  "version": "0.1.0",
```

[Changesets](https://github.com/changesets/changesets) manages this automatically